docs/source/operations/conversions/topocentric.rst

```diff
@@ -0,0 +1,104 @@
+.. _topocentric:
+
+================================================================================
+Geocentric to topocentric conversion
```

Comment:
I am missing some maths or a reference to some math somewhere on this page. There's probably something on Wikipedia or in a EPSG guidance note.

Reply:
reference to EPSG guidance note 7.2 added